python/CMakeLists.txt

```diff
@@ -55,23 +54,28 @@ target_link_libraries(_pycudaq
       cudaq-platform-default 
       fmt::fmt-header-only)
 
-install(TARGETS _pycudaq DESTINATION ${CMAKE_INSTALL_PREFIX})
-install(FILES cudaq.py DESTINATION ${CMAKE_INSTALL_PREFIX})
-install(FILES domains/chemistry.py DESTINATION ${CMAKE_INSTALL_PREFIX}/domains/)
-
+if (NOT SKBUILD)
+  install(DIRECTORY cudaq DESTINATION ${CMAKE_INSTALL_PREFIX})
```

Comment:
The package restructuring should make it such that we won't have to manually move individual files around anymore. We can just move this whole cudaq directory wherever it's needed -- in this case the build and install directories -- and all of the domains, etc. will come with it